samples/attestation/Makefile

```diff
@@ -6,6 +6,11 @@
 OE_CRYPTO_LIB := mbedtls
 export OE_CRYPTO_LIB
 
+ifndef SGX_AESM_ADDR
```

Comment:
'SGX_AESM_ADDR' should only be set at runtime. Not needed at build time. Can this block move to run:?

Reply:
As `test-samples.cmake` defines how a sample's tests run, so I moved it there after trying to export it in some targets without success.